test/e2e/protractor.config.js

```diff
@@ -0,0 +1,79 @@
+exports.config = {
+
+  //Url link for testing
+  testUrl: 'http://localhost:8000/',
+
+  sauceUser: process.env.SAUCE_USERNAME,
+  sauceKey: process.env.SAUCE_ACCESS_KEY,
+  build: process.env.TRAVIS_BUILD_NUMBER,
+
+  specs: ['./Tests/*.spec.js'],
+
+  multiCapabilities: [
+    {
+      'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+      name: 'Chrome on Linux test for build: ' + process.env.TRAVIS_BUILD_NUMBER,
+      browserName: 'chrome',
+    //  shardTestFiles: true,
+    //  maxInstances: 5,
+    //},
+    //{
+    //  'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+    //  name: 'Firefox  on Linux test for build: ' + process.env.TRAVIS_BUILD_NUMBER,
+    //  browserName: 'firefox',
+    //  shardTestFiles: true,
+    //  maxInstances: 5,
+    //},
+
+    ////@todo - uncomment out when it will ready for configurations below
+    //{
+    //  'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+    //  name: 'IE on win7 test for build: ' + process.env.TRAVIS_BUILD_NUMBER,
+    //  browserName: 'internet explorer',
+    //  //shardTestFiles: true,
+    //  //maxInstances: 5,
+    //},
+    //{
+    //  'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+    //  name: 'Safari on iPhone test for build: ' + process.env.TRAVIS_BUILD_NUMBER,
+    //  browserName: 'safari',
+    //  platformName: "iOS",
+    //  platformVersion: "7.1",
+    //  deviceName: "iPhone Simulator",
+    //  //shardTestFiles: true,
+    //  //maxInstances: 5,
+    //},
+    //{
+    //  'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+    //  name: 'Safari on OS X10.9 test for build: ' + process.env.TRAVIS_BUILD_NUMBER,
+    //  browserName: 'safari',
+    //  platform: "OS X 10.9",
+    //  //shardTestFiles: true,
+    //  //maxInstances: 5,
+    //},
+    //{
+    //  'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+    //  name: 'Chrome on Google Nexus 7 test for build: ' + process.env.TRAVIS_BUILD_NUMBER,
+    //  deviceName: 'LG Nexus 4 Emulator',
+    //  browserName: 'chrome',
+    //  //shardTestFiles: true,
+    //  //maxInstances: 5,
+    //},
+  ],
+
+  framework: 'jasmine2',
+
+  jasmineNodeOpts: {
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw ':'.<br>'jasmineNodeOpts' is not defined.